recipes/jupyterlab-system-monitor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jupyterlab-system-monitor" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-system-monitor-{{ version }}.tar.gz
+  sha256: f8d03d3ee26c84300c85e570f1a9d5f69bdcfa85e9a298f5716b67e36d0d94df
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging >=0.7.9,<0.8.0
+    - pip
+    - python >=3.6
+  run:
+    - jupyterlab >=3.0.0rc10,==3.*
+    - jupyterlab-topbar >=0.6.0,<0.7.0
+    - jupyter-resource-usage >=0.5.0,<0.6.0
+    - python >=3.6
+
+test:
+  # imports:
+  #   - jupyterlab-system-monitor
```

Comment:
It would be nice to have some sort of test that checks that the basic functionality is there (same for the other recipe). @mariusvniekerk has assured me that this will be ok for the time being.